File: ios/heihei/heihei/Session.m
//
//  Session.m
//  heihei
//
//  Created by 谢月甲 on 2017/8/3.
//  Copyright © 2017年 noear. All rights reserved.
//

#import "Session.h"
#import "Setting.h"

@implementation Session

    +(NSString*)userName
    {
        return [Setting getString:@"user_name" def:@""];
    }
    
    +(void)setUserName:(NSString*)userName{
        [Setting setString:@"user_name" value:userName];
    }
    
@end

File: android_heihei/heihei/app/src/main/java/org/noear/heihei/dao/Session.java
package org.noear.heihei.dao;

import android.content.Context;

import org.noear.heihei.App;
import org.noear.heihei.utils.SettingsUtil;

/**
 * Created by yuety on 2017/8/3.
 */

public class Session {
    public static String userName() {
        return Setting.getString("user_name","");
    }

    public static void setUserName(String userName) {
        Setting.setString("user_name", userName);
    }
}
